src/EA.Weee.Integration.Tests/Handlers/DirectRegistrant/EditOrganisationDetailsRequestHandlerIntegrationTests.cs

```diff
@@ -157,7 +157,7 @@ public static IntegrationTestSetupBuilder LocalSetup()
                 var address = fixture.Build<AddressData>().With(a => a.CountryId, country.Id).Create();
 
                 request = new EditOrganisationDetailsRequest(directRegistrant.Id, "New company name",
-                    "New trading name", address, "New brand names");
+                    "New trading name", address, "New brand names", "Reg1234");
```

Comment:
@stevelee-civica  we shouldn't be updating this request to take the company number as its a non editable field. 

Whats the underlaying cause of the bug is it being overwritten somewhere? I would suggest we create a script to remove the CompanyRegistrationNumber column from the DirectProducerSubmissionHistory and remove it from the Domain class as it shouldnt be edited from the original organisation value.

Reply:
AddSignatoryAndCompleteRequestHandler line 36
it uses the currentYearSubmission.CurrentSubmission.CompanyRegistrationNumber  which is always null

Reply:
@stevelee-civica OK lets remove that field from the database as its not needed and we shouldnt be updating it...

Reply:
should I remove the CompanyRegistrationNumber column?

Reply:
Yeah remove that from DirectProducerSubmissionHistory ...  and from  the Domain class